src/config/external-applications.md

```diff
@@ -0,0 +1,85 @@
+install the programming language's `-devel` package. This is specially relevant
+for `musl` libc users, due to pre-built binaries usually targeting `glibc`
+instead.
+
+| Language | Package Manager                | Void Package    |
+|----------|--------------------------------|-----------------|
+| Python3  | pip, anaconda, virtualenv, etc | `python3-devel` |
+| Python2  | pip, anaconda, virtualenv, etc | `python2-devel` |
+| Ruby     | gem                            | `ruby-devel`    |
+| lua      | luarocks                       | `lua-devel`     |
+
+## Restricted Packages
+
+Some packages have legal restrictions on their distribution (i.e. Discord), may
+be too large or have another condition that makes it difficult for the Void team
+to distribute. These packages are maintained but not distributed. As such, they
+must be built locally. For more information see the page on [restricted
+packages](../repositories/restricted.md).
+
+## Non-x86_64 Arch
+
+The Void build system runs on x86_64 computers, both for compiling and cross
+compiling packages. However, some packages (i.e. `libreoffice`) do not support
+cross-compilation. These packages have to be built locally on a computer running
+the same version of Void Linux as the one running where you want to use the
+package. For building packages, see the [building
+instructions](../packages/building.md).
+
+## Flatpak
+
+Flatpak is another method for installing external proprietary applications on
+Linux. For information on using Flatpak with Void Linux, see the [official
+Flatpak documentation](https://flatpak.org/setup/Void%20Linux/).
+
+> Flatpak's sandboxing will not necessarily protect you from any security and/or
+> privacy-violating features of proprietary software.
+
+However, some apps may not function properly (e.g. not being able to access the
+host system's files). To fix this, install the following packages:
+`xdg-desktop-portal`, `xdg-desktop-portal-gtk`, `xdg-user-dirs`,
+`xdg-user-dirs-gtk`, and `xdg-utils`.
+
+### Installing Themes
+
+You can install themes with Flatpak and Flatpak will automatically match your
+computer's Gtk theme to one installed with Flatpak. For example, if you use the
+theme Plata-Noir in your regular desktop environment, you can install the
+Flatpak counterpart with the command
+
+```
+$ flatpak install -y flathub org.gtk.Gtk3theme.Plata-Noir
+```
+
+You can search for additional Gtk themes with the command
+
+```
+$ flatpak search org.gtk.Gtk3theme
+```
+
+## Octave Packages
+
+Some Octave packages require external dependencies to compile and run. For
+example, to build the control package, you must install the `openblas-devel`,
+`libgomp-devel`, `libgfortran-devel`, `gcc-fortran`, and `gcc` packages.
+
+### Troubleshooting
+
+Some Flatpaks need D-Bus. Enable the dbus service with runit.
+
+Some Flatpaks require Pulseaudio to be set up properly, see
+[Pulseaudio](./media/pulseaudio.md).
```

Comment:
Suggest `dbus` instead of dbus. Since you're repeating "Some Flatpaks", I'd get these in the same paragraphs, like

"... runit. They can also require Pulseaudio..."